java/1.8/Dockerfile

```diff
@@ -19,6 +19,15 @@ RUN cd /usr/lib && \
   sbt about && \
   rm -f /tmp/sbt.zip
 
+ENV MAVEN_VERSION 3.6.2
+ENV MAVEN_HOME /usr/lib/apache-maven-$MAVEN_VERSION
+ADD http://mirror.reverse.net/pub/apache/maven/maven-3/3.6.2/binaries/apache-maven-$MAVEN_VERSION-bin.zip /tmp/maven.zip
```

Comment:
there's another 3.6.2 to replace with `$MAVEN_VERSION` in this string

Reply:
OH! Indeed!